packages/as-proba/assembly/probability/binomial.ts

```diff
@@ -27,12 +28,11 @@ export class Binomial extends Sampler implements Drawer {
   }
 
   /**
-   * Returns the probability of an event.
+   * This method calculates the probability of a given event k using the mass probability function
+   * of the binomial distribution an returns it.
```

Comment:
You can be more straightforward here:
```suggestion
   * Computes the probability of a given event `k`.
```